src/pages/Login/components/RegistrationComponent/RegistrationComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Button, ButtonTheme } from '../../../../components/Button';
+import { Card } from '../../../../components/Card';
+import { Input, InputTypeEnum } from '../../../../components/Input';
+
+import { RegistrationComponentProps } from './types';
+
+import styles from './RegistrationComponent.css';
+
+export const RegistrationComponent = (props: RegistrationComponentProps) => {
+  const { switchForm } = props;
+  return (
+    <Card title={'Registration'}>
```

Comment:
❗️
Когда нет никакой логики, нужно явно это указывать. Если где-то ещё такое юзается, там тоже надо поправить
```suggestion
    <Card title="Registration">
```

Reply:
исправил